packages/maskbook/src/plugins/RedPacket/constants.ts

```diff
@@ -7,8 +7,8 @@ export const RedPacketPluginID = 'com.maskbook.red_packet'
 export const RED_PACKET_CONSTANTS = {
     HAPPY_RED_PACKET_ADDRESS: {
         [ChainId.Mainnet]: '0x26760783c12181efa3c435aee4ae686c53bdddbb',
```

Comment:
mainnet doesnt update, does it mean mainnet still using the old contract?

Reply:
ye